examples/use_case_repository/use_case_repository/guides/_TEMPLATE.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Snowflake to S3 ETL with Dagster"
+description: "This use case demonstrates how to transfer data from Snowflake to Amazon S3 using Dagster. The objective is to automate the extraction of data from Snowflake and store it in S3 for further processing or archival."
+tags: ["snowflake", "s3" ]
+---
+
+# [Insert a Use Case Name that has high SEO Value here]
+
+Provide a brief overview of the use case, including its objectives and the main problem it addresses. All use cases must use Dagster to accomplish tasks.
+
+---
+
+## What You'll Learn
+
+You will learn how to:
+
+    - Define a Dagster asset that extracts data from an external source and writes it to a database
+    - Add other bullets here
+    - ...
```

Comment:
```suggestion
- Define a Dagster asset that extracts data from an external source and writes it to a database
- Add other bullets here
- ...
```

Nit - let's removing the spacing in the template to prevent it from getting wonky when people edit it